src/TypeScript.UnitTests/NodeJSLocator/NodeLocationsProviderTests.cs

```diff
@@ -32,7 +36,22 @@ public class NodeLocationsProviderTests
         [TestMethod]
         public void MefCtor_CheckIsExported()
         {
-            MefTestHelpers.CheckTypeCanBeImported<NodeLocationsProvider, INodeLocationsProvider>(null, null);
+            MefTestHelpers.CheckTypeCanBeImported<NodeLocationsProvider, INodeLocationsProvider>(null, new []
+            {
+                MefTestHelpers.CreateExport<SVsServiceProvider>(Mock.Of<IServiceProvider>()),
+                MefTestHelpers.CreateExport<ILogger>(Mock.Of<ILogger>())
+            });
+        }
+
+        [TestMethod]
+        public void Ctor_InitializesCorrectProviders()
```

Comment:
`Ctor_InitializesCorrectProviders_InPriorityOrder` would be more explicit that the order is important.